client/my-sites/checkout/controller.jsx

```diff
@@ -47,6 +47,34 @@ import { TRUENAME_COUPONS } from 'calypso/lib/domains';
 
 const debug = debugFactory( 'calypso:checkout-controller' );
 
+export function checkoutSiteless( context, next ) {
```

Comment:
Thinking about renaming this method to `checkoutJetpack`, and moving all the code we added for the Logged Out Visitor Checkout project from the `checkout` controller to this one. Currently, the `checkout` controller handles many flows/conditions and is becoming very hard to grok/maintain.